gtfs/Governance/change-process.md

```diff
@@ -0,0 +1,383 @@
+    
+2. **Voting Process**  
+     
+    * [Contributors](roles.md/#contributors) must vote in the comment section of the Pull Request.  
+    
+3. **Editing and Canceling**  
+     
+    * The [Advocate](roles.md/#advocate) can edit the proposal only for editorial purposes during the voting period.  
+    * The [Advocate](roles.md/#advocate) can cancel the vote at any time.  
+    
+4. **End of Vote Announcement**  
+     
+    * The [Advocate](roles.md/#advocate) announces the end of the vote in the Pull Request comment section and includes the result.  
+    * The [Advocate](roles.md/#advocate) also announces the end of the vote in the [GTFS Changes mailing list](https://groups.google.com/g/gtfs-changes) discussion thread, including the result.
+
+5. **Failed Vote**  
+     
+    * If the vote fails, the [Advocate](roles.md/#advocate) can choose to either:  
+        1. Adjust proposal based on actionable feedback provided and call another vote,  
+        2. Go back to the discussion step ([step 2.2](#step-22-pull-request-discussion)) and redefine the scope, or  
+        3. Abandon the proposal.  
+    * The [Advocate](roles.md/#advocate) must announce their decision in the Pull Request comment section and in the [GTFS Changes mailing list](https://groups.google.com/g/gtfs-changes) discussion thread.
+
+**<ins>Requirements</ins>** 
+
+The vote must meet the following conditions:
+
+| Requirement | Details |
+| :---- | :---- |
+| **Consensus Principle** | The vote is based on **a qualified majority** (80% majority) |
+| **Vote Passing Criteria** | The vote passes when 80% or more of contributors vote **\+1**. |
+| **Vote Failing Criteria** | The vote fails when more than 20% of contributors vote **\-1**. |
+| **Voting Format** | Votes must be formatted as follows: <br>- *“+1 or \-1, Organization Name, Contributor Type (Consumer, Producer, or General Contributor), Link to Produced Feed or Consuming Application”* |
+| **Voting Against** | Contributors providing a negative vote (-1) must give actionable feedback. <br>- Actionable feedback is practical and constructive, providing concrete observations or suggestions to help solve the identified issue:  <br> - “*This proposal does not respect the backward compatibility principle of GTFS and we propose to create a separate file instead.*” |
+| **Minimum Votes** | At least 5 votes must be cast. |
+| **Participant Composition** | At least 2 consumers and 2 producers must participate in the vote. <br>- Each of these contributors can only be considered either a Consumer or Producer even if they are able to represent both roles. <br>- First adopters who tested the proposal can vote and count toward this requirement, provided they are not acting as the Advocate for the proposal. |
```

Comment:
Just to confirm, do the 2 consumer + 2 producer votes all need to be +1 votes?  Or could you have 5 votes where 2 consumers vote but one of the consumer votes is a -1?